pkg/templates/machinecontroller/helper.go

```diff
@@ -22,36 +22,43 @@ import (
 
 	"github.com/pkg/errors"
 
+	"k8c.io/kubeone/pkg/addons"
 	"k8c.io/kubeone/pkg/clientutil"
 	"k8c.io/kubeone/pkg/state"
+	"k8c.io/kubeone/pkg/templates/resources"
 
 	clusterv1alpha1 "github.com/kubermatic/machine-controller/pkg/apis/cluster/v1alpha1"
 
 	corev1 "k8s.io/api/core/v1"
 	errorsutil "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/util/retry"
 	dynclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// machineController related constants
+const (
+	mcNamespace     = metav1.NamespaceSystem
+	mcAppLabelKey   = "app"
+	mcAppLabelValue = "machine-controller"
+)
+
+// machineControllerWebhook related constants
+const (
+	whName          = "machine-controller-webhook"
+	whAppLabelKey   = mcAppLabelKey
+	whAppLabelValue = whName
+	whNamespace     = mcNamespace
+)
```

Comment:
Don't we define some of those consts in the resources package as well? Maybe we could reuse some of the consts from that package instead of redefining them.

Reply:
good point